CodeStarterCamp_Week2/main.swift

```diff
@@ -6,7 +6,40 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
+let myLottoNumbers: [Int] = [4, 31, 11, 7, 28, 45]
+
+var lottoNumbers: Set<Int> = Set<Int>()
+
+func lottoPlease() -> Set<Int>{
```

Comment:
귀여운 네이밍이네요 🥰
다만, [API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/)에 맞는 네이밍은 아닌 것 같습니다 😅

함수명을 지을 때는 사이드 이팩트에 따라 지어주는 것이 좋다고 하는군요!
이를 참고해보면, 동사로 시작하는 것이 좋아보입니다.
함수의 로직인 로또 번호를 생성하는 것을 잘 나타낼 수 있는 네이밍을 한번 고민해볼까요? 😁

아 그리고 띄어쓰기 오타가 있네요! 👀
```suggestion
func lottoPlease() -> Set<Int> {
```